docs/source/installation/migration.rst

```diff
@@ -0,0 +1,127 @@
+=========
+Migration
+=========
+
+Since v1.0.0 has many breaking changes, we need to do some preparation to migrate your existing v0.4.x version of yatai.
+
+.. note:: If your yatai is below v0.4.6, you must upgrade it to v0.4.6 first.
+
+Breaking changes
+----------------
+
+* Split into two modules
+
+  * yatai
+
+    Dashboard and bento registry
+
+  * yatai-deployment
+
+    Deploy bento to Kuberenetes
+
+* Remove all yatai component operators
+
+  In v0.4.x, yatai component operators will automatically install dependencies of the component. But in v1.0.0, we removed this feature.
+
+  Some things will changes:
+
+  * No more integration of logging and monitoring
+
+    Because we removed all yatai component operators, yatai now does not automatically integrate logging and monitoring. See the `Observability </observability/index.html>`_ documentation for observability configuration.
+
+Migration steps
+---------------
+
+1. Backup PostgreSQL data
+"""""""""""""""""""""""""
+
+.. note:: If you use the external PostgreSQL, you need to skip this step.
```

Comment:
```suggestion
.. note:: If you use an external PostgreSQL, you can to skip this step.
```

Reply:
@ssheng If using an external database, this must be skipped